### Assembly

The gneome assembly was created by the [Iwate Biotechnology Research
Center](http://europepmc.org/articles/PMC5604175). The TDr96\_F1 line
used for whole genome sequencing was selected from F1 progeny obtained
from an open-pollinated *D. rotundata* breeding line (TDr96/00629) grown
under field conditions on the experimental fields of the International
Institute of Tropical Agriculture (IITA) in Nigeria. Total DNA from leaf
tissue was used for construction of paired-end and mate-pair libraries
(2, 3, 4, 5, 6, 8, 20, and 40 kb insert). These were sequenced using the
Illumina MiSeq and HiSeq2500 platforms. In addition, 30,750 BAC clones
corresponding to 3,072 Mb of sequence and 5.4Ã— genome coverage were
constructed. Of these, 9984 clones were used for BAC-end sequencing,
with paired-end reads. A total of 85.14 Gb of sequence data was
generated, representing \~149.4x coverage of the 570 Mb genome size
estimated by flow cytometry.

Reads were assembled following the ALLPATHS-LG pipeline, using
paired-end and mate-pair data, with additional scaffolding carried out
using SSPACE and the BAC-end reads. Scaffolds were anchored and ordered
into 21 pseudo-chromosomes using a genetic map generated from 150 F1
individuals, obtained from a cross between TDr97/02627 (P1:Female) and
TDr99/02627 (P2:Male) breeding lines using RAD-tags as linkage markers.
The 21 pseudo-chromosomes represent 76.5 % of the total scaffolds, with
a size of \~454 Mb.
