### Assembly

The genome assembly was created by the Iwate Biotechnology Research Center. 
The TDr96_F1 line used for whole genome sequencing was selected from F1 progeny 
obtained from an open-pollinated D. rotundata breeding line (TDr96/00629) 
grown under field conditions on the experimental fields of the International 
Institute of Tropical Agriculture (IITA) in Nigeria. Total DNA from fresh 
leaves was sequenced with a PromethION sequencer (Oxford Nanopore Technologies), 
representing 36.6x genome coverage of the 570 Mbp genome size estimated by flow 
cytometry.

The generated long reads were assembled by Flye, and the assembled contigs 
were polished by Pilon using Illumina short reads in the previous study (1). 
Then, the duplicated contigs were removed by Purge Haplotigs, and the retained 
contigs were polished by Pilon again. The contigs were anchored and ordered 
into 20 pseudo-chromosomes using a genetic map generated from the whole genome 
sequences of 156 F1 individuals, obtained a cross between TDr04/219 (P1: Female) 
and TDr97/777 (P2: Male) breeding lines. The 20 pseudo-chromosomes (named 1 to 20) 
represent 84.9% of the total contigs, with a size of 492 Mbp. The remaining are 
unanchored linkage groups and contigs.
