### Annotation

Illumina RNA-seq reads were assembled using Trinity v2.5.1 with the de novo mode and the genome-guided mode, respectively. The resulting transcriptome assemblies and the PacBio Iso-Seq full-length cDNA sequences were used as transcript evidence. Ab initio gene predictions were performed using Augustus v3.2.3, GeneMark-ET v4.33 and SNAP v2006-07-28. Proteins from SwissProt and from Arabidopsis, watermelon, cucumber and melon were aligned to the genome using Spaln v2.1.4, and the resulting alignments were used as protein homology evidence. Maker v3.01.02 was then run to generate high-confidence gene models by integrating ab initio predictions, transcript mapping evidence and protein homology evidence.

Repeats were annotated with the [Ensembl Genomes repeat feature pipeline](http://plants.ensembl.org/info/genome/annotation/repeat_features.html). There are: 841,860 Low complexity (Dust) features, covering 38 Mb (10.3% of the genome); 214,000 RepeatMasker features (with the nrTEplants library), covering 71 Mb (19.3% of the genome); 99,758 RepeatMasker features (with the REdat library), covering 34 Mb (9.2% of the genome); 284,068 Tandem repeats (TRF) features, covering 19 Mb (5.3% of the genome).
