### Annotation

Sanger-derived EST resources and a substantial amount of new RNA
sequencing reads (727 million reads from 11 tissues and developmental
stages) where combined with homology-based and de novo gene prediction
approaches. A total 43,627 transcript assemblies were obtained PERTRAN
and PASA. Loci were identified by transcript assembly alignments and/or
EXONERATE alignments of peptides from *Arabidopsis*, poplar, *Medicago
truncatula*, grape and rice peptides to the repeat-soft-masked genome.
Gene models were predicted by the homology-based predictors FGENESH+,
FGENESH\_EST and GenomeScan. The highest scoring predictions for each
locus were selected using multiple positive factors. Selected gene
predictions were improved by PASA, including by adding UTRs, correcting
splicing and adding alternative transcripts. PASA-improved gene model
peptides were subjected to peptide homology analysis with the
above-mentioned proteomes to obtain Cscore values and peptide coverage.
Selected gene models were subjected to Pfam analysis, and gene models
whose encoded peptide contained more than 30% Pfam transposon element
domains were removed. The resulting annotation includes 28134
protein-coding loci. Most of these genes (91%) were retained in synteny
blocks with soybean.
