### Annotation

Protein-coding genes were predicted using de novo and protein
homology-based approaches. Genscan v1.0, Augustus v2.5.5, GlimmerHMM
v3.0.1, GeneID v1.3, and SNAP were performed for de novo gene
prediction, while homologous peptides from the A. thaliana (TAIR 10),
Oryza sativa (Nipponbare, IRGSP-1.0), Theobroma cacao (Phytozome v12.1),
and C. sinensis (Phytozome v12.1) genomes were aligned to our assembly
to identify the homologous genes with GeMoMa v1.4.2. RNA-Seq reads were
assembled using Trinity, and the resulting unigenes were aligned to the
repeat-masked assemblies using BLAT, and subsequently, the gene
structures of BLAT alignment results were modeled using PASA. Then,
protein-coding regions were identified with TransDecoder v3.0.1 and
GeneMarkS-T, respectively. Consensus gene models were generated by
integrating the de novo predictions and protein alignments using
EVidenceModeler.
