### Variation

[**Data from CerealsDB**](http://europepmc.org/abstract/MED/27342803)

31,779 (out of 35,143) markers from the **35K Axiom SNP array** and
768,664 (out of 819,570) markers from the **820K Axiom SNP array** were
aligned to the assembly. This was done by
[CerealsDB](http://europepmc.org/abstract/MED/27342803) using 101bp
sequences with the SNP located centrally at position 51. Blastn E-value
cutoff was set to 1e-05. The top three hits were parsed and compared to
genetic map data. In cases where two or more of the top three hits had
an identical score, the hit agreed with the genetic map was selected. In
cases of no genetic map information for a particular SNP then the top
hit was selected. Some of the markers failed to align to the assembly or
only aligned to the chrUn contigs (unassigned contigs), or couldn\'t be
unambiguously assigned to a particular chromsome so they were removed.

The 35K set includes genotype data from 1,963 samples while the 820K has
475 samples. In cases where a marker belongs to both sets, genotype data
from both 820K and 35K samples will be displayed. \[5\]

[**EMS Mutation data**](http://europepmc.org/abstract/MED/28096351)

EMS-type variants from sequenced tetraploid (cv 'Kronos') and hexaploid
(cv 'Cadenza') TILLING populations. Mutations were called on the IWGSC
RefSeq V1.0 assembly using the [Dragen
system.](https://file.scirp.org/Html/2-1370287_74603.htm)

-   3.9 million Kronos mutations
-   8.2 million Cadenza mutations

Researchers and breeders can search this database online, identify
mutations in the different copies of their target gene, and request
seeds to study gene function or improve wheat varieties. Seeds can be
requested from the [UK
SeedStor](https://www.seedstor.ac.uk/shopping-cart-tilling.php) or from
the US based [Dubcovsky
lab](http://dubcovskylab.ucdavis.edu/wheat-tilling).This resource was
generated as part of a joint project between the University of
California Davis, Rothamsted Research, The Earlham Institute, and the
John Innes Centre. \[2\]

**Inter-Homoeologous Variants**

3.6 million Inter-Homoeologous Variants (IHVs) called by alignments of
the A,B and D component genomes where added as SNP markers.

**SIFT scores**

[SIFT](http://sift.bii.a-star.edu.sg/) predicts whether an amino acid
substitution affects protein function based on sequence homology and the
physical properties of amino acids. SIFT can be applied to naturally
occurring nonsynonymous polymorphisms and laboratory-induced missense
mutations.

SIFT scores and predictions (whether it is \'tolerated\' or
\'deleterious\') have been calculated for all missense variants across
all bread wheat variation datasets.

**KASP markers**

KASP markers designed to be genome-specific with PolyMarker\[6\] are
displayed for the EMS-type variants. For details on how to interpret the
annotation and details please visit <http://www.polymarker.info/about>.
