### Regulation and sequence alignment

RNA-seq data, ESTs and UniGene datasets have also been aligned to the
*Triticum urartu* genome:

-   *Triticum urartu* 454 RNA-seq data, from the ENA study
    [SRP002455](http://www.ebi.ac.uk/ena/data/view/SRP002455), and
    published by [Akhunova *et
    al*.](http://europepmc.org/abstract/MED/20849627), were aligned
    using [STAR](https://code.google.com/p/rna-star/).
-   Wheat UniGene cluster sequence data were aligned using
    [Exonerate](http://www.ebi.ac.uk/%7Eguy/exonerate/), following the
    standard Ensembl pipeline.
-   All publicly available wheat EST data were aligned using
    [STAR](https://code.google.com/p/rna-star/).

#### Analysis of the bread wheat genome using comparative whole genome shotgun sequencing

The wheat genome assemblies previously generated by
[Brenchley](http://europepmc.org/abstract/MED/23192148) *[et
al.](http://europepmc.org/abstract/MED/23192148)* have been aligned to
the bread wheat survey sequence, *Brachypodium*, barley and the wild
wheat progenitors (*Triticum urartu* and *Aegilops tauschii*).
Homoeologous variants inferred between the three wheat genomes (A, B,
and D) are displayed in the context of the gene models of these five
genomes.

Sequences of diploid progenitor and ancestral species permitted
homoeologous variants to be classified into two groups, 1) SNPs that
differ between the A and D genomes (where the B genome is unknown) and,
2) SNPs that are the same between the A and D genomes, but differ in B.

#### Transcriptome assembly in diploid einkorn wheat *Triticum monococcum*

Genome-wide transcriptomes of two *Triticum monococcum* subspecies were
constructed by [Fox *et
al*](http://europepmc.org/abstract/MED/24821410), the wild winter wheat
*T. monococcum ssp. aegilopoides* (accession G3116) and the domesticated
spring wheat *T. monococcum ssp. monococcum* (accession DV92) by
generating *de novo* assemblies of RNA-seq data derived from both
etiolated and green seedlings. Assembled data is available from the
[Jaiswal lab](http://jaiswallab.cgrb.oregonstate.edu/genomics/wheat) and
raw reads are available from INSDC projects
[PRJNA203221](http://www.ebi.ac.uk/ena/data/view/PRJNA203221) and
[PRJNA195398](http://www.ebi.ac.uk/ena/data/view/PRJNA195398).

The de novo transcriptome assemblies of DV92 and G3116 represent 120,911
and 117,969 transcripts, respectively. They were mapped to the [bread
wheat,](http://Triticum%20aestivum)
[barley](http://plants.ensembl.org/Hordeum_vulgare) and *Triticum
urartu* genomes using [STAR](https://code.google.com/p/rna-star/).
