### Annotation

Homology-based and de novo methods, as well as RNA-seq data, were used
to predict genes. GLEAN was used to consolidate results. Protein
sequences of several plants were aligned with TBLASTN and genBLASTA
against the matching genomic sequence by using GeneWise for accurate
spliced alignments. Next, the de novo gene-prediction methods GlimmerHMM
and Augustus were used to predict protein-coding genes, with parameters
trained for O. europaea var. sylvestris, A. thaliana, S. indicum, S.
tuberosum, and V. vinifera. A total of 50,684 protein-coding genes were
predicted, of which 47,124 genes (93%) were confirmed by transcriptome
data. A total 31,245 genes were located on the anchored chromosomes.

\
Repeats were annotated with the [Ensembl Genomes repeat feature
pipeline](http://plants.ensembl.org/info/genome/annotation/repeat_features.html). There
are: 1619023 Low complexity (Dust) features, covering 160 Mb (14.0% of
the genome); 1281455 RepeatMasker features (with the REdat library),
covering 261 Mb (22.9% of the genome); 8004 RepeatMasker features (with
the RepBase library), covering 1 Mb (0.1% of the genome); 513948 Tandem
repeats (TRF) features, covering 374 Mb (32.8% of the genome).
