### Annotation

[Gene models were predicted by Wageningen University by several tools
and integrated using EVM, resulting in 11,700 predicted protein-coding
genes. RNASeq data of samples from \>20 conditions were used for gene
model validation. The gene models were then revised by an comprehensive
manual review undertaken by a community of \~25 scientists and supported
by Ensembl Genomes3, and made availabe in release 34. Functional
annotation and comparative analyses were conducted through the Ensembl
Genomes pipeline.]{style="font-size:12px"}

[The following members of the Botrytis community participated in the
structural curation of gene models: Matthias Hahn, Sylvia Thoms, Juliane
Schurig, Nathalie Knüppel (University of Kaiserslautern, Germany);
Ernesto Benito (University of Salamanca, Spain); Daniela Santander,
Vinicio Armijos (Universidad de Las Américas, Quito, Ecuador); Celedonio
Gonzalez, Nelida Brito, Mario Gonzalez (University of La Laguna, Spain);
Rita Milvia DeMiccolis Angelini, Caterina Rotolo (University of Bari,
Italy); Nathalie Poussereau, Cecile Ribot, Amelie Devallee, Christine
Rascle, MarieJoseph Gagey, Mathias Choquer, Christophe Bruel (University
Claude Bernard Lyon 1, France); Adeline Simon, Muriel Viaud, Jaafar
Kilani (Versailles-Grignon Research Centre, INRA, France); Andreas
Mosbach, Gabriel Scalliet (Syngenta, Switzerland); Paulo Canessa
(Universidad Andres Bello, Santiago, Chile); Jan van Kan (Wageningen
University, the Netherlands). Their tireless efforts to improve the
annotations to the best of their capacity are gratefully
acknowledged.]{style="font-size:12px"}

[Non coding RNA genes have been annotated using tRNAScan-SE (Lowe, T.M.
and Eddy, S.R. 1997), RFAM (Griffiths-Jones et al 2005), and RNAmmer
(Lagesen K.,et al 2007); additional analysis tools have also been
applied.]{style="font-size:12px"}
