[A variation database has been constructed by comparing this genome with
]{style="font-family: 'Luxi Sans', Helvetica, Arial, Geneva, sans-serif; line-height: 16px;"}*[[Puccinia
graminis f. sp.
tritici]{style="line-height: 16px;"}]{face="Luxi Sans, Helvetica, Arial, Geneva, sans-serif"
style="font-family: 'Luxi Sans', Helvetica, Arial, Geneva, sans-serif;"}*[
sequence read data
(]{style="font-family: 'Luxi Sans', Helvetica, Arial, Geneva, sans-serif; line-height: 16px;"}[[[study
PRJNA253722](http://www.ebi.ac.uk/ena/data/view/PRJNA253722)). Reads
were]{style="line-height: 16px;"}]{face="Luxi Sans, Helvetica, Arial, Geneva, sans-serif"
style="font-family: 'Luxi Sans', Helvetica, Arial, Geneva, sans-serif;"}[
mapped against the
]{style="font-family: 'Luxi Sans', Helvetica, Arial, Geneva, sans-serif; line-height: 16px;"}*Puccinia
graminis*[ Ug99
]{style="font-family: 'Luxi Sans', Helvetica, Arial, Geneva, sans-serif; line-height: 16px;"}[genome
using BWA (Li H.and Durbin R.Bioinformatics 2010) and variations called
using SAMtools (Li H.et.al Bioinfomratics
2009).]{style="font-family: 'Luxi Sans', Helvetica, Arial, Geneva, sans-serif; line-height: 16px;"}
