
PacBio long-reads were assembled with the diploid FALCON-Unzip assembler, producing a primary set of contigs and a set of linked haplotigs that represented the alternative genome structures of the primary contigs. The primary assembly was upgraded using FinisherSC and then polished, together with the haplotigs, using Arrow and Pilon tools. A total of 333 Mbp of the fig genome sequence, corresponding to ~95% of the estimated size, were obtained, and the primary contigs had a mean contig size of 368Kbp and N50 of 823Kbp. The mean contig size of the haplotigs was 58 Kbp and the N50 was 89 Kbp. The final phased genome is ~333 Mbp in size, of which 80% has been anchored to 13 chromosomes. After excluding fungal and bacterial contamination, we measured the core gene completeness using BUSCO software. BUSCO recovered 1283 of the 1375 (93.3%) highly conserved Embryophyta core genes, of which 1177 (85.6%) were complete and single-copy and 106 (7.7%) were complete and duplicated, 35 genes (2.5%) were fragmented, and 57 (4.2%) missing. 
