### Annotation

Gene models were annotated on the MorexV3 pseudomolecules using the same transcriptomic 
resources as used for MorexV2/TRITEX, but with an improved version of the PGSB annotation pipeline, 
which is also able to call isoforms and UTRs. A total of 81,687 genes with 83,990 transcripts were identified. 
Of these, 35,827 were classified as high-confidence (HC) genes. Among all gene models, 
98.6% of BUSCO models were retrieved. Moreover, 91% of V3 gene models had no ambiguous bases 
in their 100 kb flanking sequence compared to only 0.7% in MorexV2. The coding sequences of 35,260 
(98.4%) Morex V3 HC gene models had near-complete alignments (≥95% alignment coverage, ≥99% identity) 
to the V2 pseudomolecules.

Moreover, models corresponding to the barley gene reference transcript dataset (BaRTv1.0) are also supported.
These were derived from the analysis of 22 RNA-seq experiments covering 843 separate samples.

Repeats were annotated with the [Ensembl Genomes repeat feature pipeline](http://plants.ensembl.org/info/genome/annotation/repeat_features.html). There are: 3431114 Low complexity (Dust) features, covering 158 Mb (3.7% of the genome); 1681147 RepeatMasker features (with the nrTEplants library), covering 3212 Mb (76.0% of the genome); 2100078 Tandem repeats (TRF) features, covering 325 Mb (7.7% of the genome); Repeat Detector repeat length: 3422Mb (80.9% of the genome).
