*AcolM1* assembly
-----------------

*An. coluzzii* Mali-NIH, formerly *Anopheles gambiae* M form, was
sequenced by the Washington University School of Medicine Genome
Sequencing Center (Lawniczak et al. 2010, PMID: 20966253). DNA samples
derived from whole mosquitoes were provided by the University of Notre
Dame and MR4. BAC libraries were provided by the Clemson University
Genomics Institute (CUGI), and are available through CUGI or MR4. The
number of traces per genome was \~2.755 million M sequence reads
deposited in the NCBI Trace Archives. For the M project, 94% of the
traces were from plasmids (5-6kb inserts), 4% from fosmids (40kb
inserts), and 2% from BACs. Based on the source DNA of these libraries,
94% of reads were generated from males, resulting in lower X-chromosome
sequence coverage relative to autosomes.

Whole genome shotgun (WGS) sequences were assembled de novo for each
genome by both sequencing centers: at WUGSC using the PCAP assembler,
and at JCVI using the Celera assembler
([http://wgs-assembler.sf.net](http://wgs-assembler.sf.net/)). WUGSC
assemblies based on the original PCAP algorithm were nearly twice the
expected \~260 Mb size. This outcome reflected considerable numbers of
high quality base discrepancies (polymorphisms), owing to relatively
high allelic variation in the non-isogenic genome samples. Although a
modification of PCAP (Pcap.rep.poly) resulted in smaller assembled
genome sizes, the Celera assembler algorithms specifically developed to
accommodate heterozygosity gave improved assemblies. By mutual
agreement, the JCVI assemblies (available via GenBank accessions
ABKP00000000 and ABKQ00000000) and served as the basis for the
VectorBase genome browsers.
