*LlonJ1* assembly
-----------------

Three types of WGS libraries were used to produce these *Lutzomyia*
*longipalpis* sequencing data: a 454 Titanium fragment library and
paired end libraries with 3 kb and 8 kb inserts. The 454 data (11.5
million reads; \~24.4x coverage) was derived from the same individual
while mate pair reads (7.4 million 3kb reads, 9.6X; 3.7 million 8kb
reads, 4.9X) were derived from a pool of individuals. In total about
22.6 million reads were generating representing 38.9x coverage of this
sand fly genome.

The *Llon\_1.0* assembled draft genome sequence was built from the data
described above using the Celera CABOG assembler (version 6.1,
2010/03/22). Next, these initial results were used as a backbone for
longer superscaffolds using Baylor\'s ATLAS-link. Finally, discernible
gaps filled with ATLAS-gapfill and ATLAS-gapmerge. The final assembly
includes these superscaffolds, which can be ordered and oriented with
respect to each other, and isolated sequences that could not be linked
(single contig scaffolds or singletons from the original CABOG
assembly).

The total length of all contigs is 142.7 Mb; however, the total span of
the assembly is 154.2 Mb after gaps are included. The N50 of the contigs
is 7.5 kb and the N50 of the scaffolds is 85.1 kb. Both the assembly and
the description of the positions and orientations of contigs (AGP) are
available from the Sand fly section of the BCM-HGSC web site at:
[www.hgsc.bcm.tmc.edu](http://www.hgsc.bcm.tmc.edu/).
