### Assembly

The genome of pineapple variety F153 was sequenced and assembled using
400× Illumina reads, 2× Moleculo synthetic long reads, 1× 454 reads, 5×
PacBio single-molecule long reads and 9,400 BACs. Because of
self-incompatibility, pineapple has high levels of heterozygosity and is
cultivated through clonal propagation. To overcome the difficulties in
assembling this highly heterozygous genome, we applied a genetic
approach to reduce the complexity of the genome using a cross between
F153 and the CB5 A. bracteatus accession. The haploid genomes of the F1
plants were used for haplotype phasing to improve the assembly. Using
this approach substantially improved the initial Illumina-only assembly
and spanned 382 Mb, 72.6% of the estimated 526 Mb of the pineapple
genome.\
A more recent CB5 chromosome level assembly and the distribution of
Gypsy LTR-RT elements around centromeres were used to further improve
the F153 assembly (F153 v.7), in which LG01 was separated into AccChr1
and AccChr24, while LG24 and LG25 were linked together into AccChr25.
