### Annotation
A high confidence gene set of 38,919 gene models with good protein or EST support was constructed for L. sativa by merging gene models from different prediction pipelines. These gene models have average coding lengths of 1.05 kb and 4.5 exons per gene, similar to those in other sequenced plant genomes. The average intergenic distance of 39.5 kb places L. sativa between Nicotiana tomentosis and Capsicum annum consistent with a direct correlation between intergenic distance and genome size. Out of the total number of predicted genes, 29,681 (76.27%) genes had similarity to Arabidopsis TAIR10 annotations and 28,951 (74.3%) were annotated using InterProScan521. Annotation using KEGG22 database yielded information for 7,553 L. sativa gene predictions. The combined data sets provided functional annotation for 31,348 (80.5%) gene models.(1)


<!-- This browser is based on data from
[Araport11](https://www.araport.org/data/araport11) gene annotation, a
comprehensive reannotation of the TAIR10 genome, released June, 2016.
Details of the structural and functional annotation steps to generate the Araport11
protein-coding gene set as well as consolidation and annotation of
non-coding RNAs are described in <https://doi.org/10.1111/tpj.13415>.

Repeats were annotated with the [Ensembl Genomes repeat feature
pipeline](http://ensemblgenomes.org/info/data/repeat_features). There
are: 429457 Low complexity (Dust) features, covering 17 Mb (7.3% of the
genome); 74964 Repeats (ENA) features, covering 107 Mb (47.2% of the
genome); 70178 RepeatMasker features (with the REdat library), covering
21 Mb (9.2% of the genome); 2198 RepeatMasker features (with the RepBase
library), covering 0 Mb (0.1% of the genome); 147273 Tandem repeats
(TRF) features, covering 13 Mb (5.6% of the genome). -->


