### Assembly

[Illumina GA and HiSeq 2000 Sequencing
system](https://europepmc.org/abstract/MED/22057054) were used to
sequence 11 small-insert (180--800 bp) and 11 large-insert (2--20 kb)
libraries [1]. This generated a total of 237.2 Gb of paired-end reads,
ranging from 50--100 bp. Filtering and correction of the sequence data
for very small and/or bad-quality sequences yielded 130.7 Gb of
high-quality sequence, ∼163.4×
coverage of the pigeonpea genome. Analysis of sequence data for GC
content indicated a similar GC content distribution in the genomes of
pigeonpea and soybean. Additionally, a set of 88,860 bacterial
artificial chromosome (BAC) end sequences were generated using Sanger
sequencing from two BAC libraries (69,120 clones) by using the HindIII
(34,560 clones) and BamHI (34,560 clones) restriction enzymes.

SOAPdenovo was used to assemble 605.78 Mb
of the pigeonpea genome *de novo*, generating a sequence with a contig
N50 of 21.95 kb, and longest contig length of 185.39 kb. This was then
improved by using both the paired-BAC end sequences (41,302) that passed
after filtering through RepeatMasker, and a genetic map comprising 833
marker loci. This increased N50 to 516.06 kb (longest scaffold in
chromosome level of 48.97 Mb) . The draft genome assembly has <5.69%
(~34 Mb) unclosed gaps. These
analyses showed that mapped genetic loci provide additional information
for assembling superscaffolds, especially in regions in which scaffolds
were not large enough to cross the repeat rich regions. The generated
chromosome-scale scaffolds can be considered as 'pseudomolecules'. The
estimated pigeonpea genome size, based on K-mer statistics, is 833.07
Mb, suggesting that the assembly captures 72.7% of the genome in the
genome scaffolds. If only the 6,534 scaffolds \>2 kb are considered, the
assembly spans 578 Mb with an N50 of 0.58 Mb.
